<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOne">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnOne{	
	background-color: rgb(194, 64, 194);
	color: black;
	border-radius:25px;
}
QPushButton#btnOne:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(194, 64, 194), stop:1  rgb(194, 64, 194));
}
QPushButton#btnOne:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(194, 64, 194);
}</string>
    </property>
    <property name="text">
     <string>Yönetici</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTwo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnTwo{	
	background-color: rgb(141, 209, 104);
	color: black;
	border-radius:25px;
}
QPushButton#btnTwo:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(141, 209, 104), stop:1  rgb(141, 209, 104));
}
QPushButton#btnTwo:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(141, 209, 104);
}</string>
    </property>
    <property name="text">
     <string>Öğretim Üyesi</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/images/logo3.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1839</x>
      <y>40</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnQuit{	
	background-color: transparent;
	color: rgb(0, 0, 0);
	border-radius:5px;
}
QPushButton#btnQuit:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(255, 65, 65), stop:1  rgb(255, 65, 65));
}
QPushButton#btnQuit:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(255, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/feather/x.svg</normaloff>:/icons/feather/x.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThree">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnThree{	
	background-color: rgb(206, 137, 0);
	color: black;
	border-radius:25px;
}
QPushButton#btnThree:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(206, 137, 0), stop:1  rgb(206, 137, 0));
}
QPushButton#btnThree:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(206, 137, 0);
}</string>
    </property>
    <property name="text">
     <string>Öğreni Girişi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUsername">
    <property name="geometry">
     <rect>
      <x>1509</x>
      <y>640</y>
      <width>331</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);

</string>
    </property>
    <property name="placeholderText">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePassword">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>770</y>
      <width>330</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(105,118,132,255);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>890</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnLogin{	
	background-color:rgb(0, 255, 127);
	color: black;
	border-radius:15px;
}
QPushButton#btnLogin:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(0, 255, 127), stop:1  rgb(0, 255, 127));
}
QPushButton#btnLogin:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(0, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>GİRİŞ</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseTitle">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>570</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>logo</zorder>
   <zorder>btnOne</zorder>
   <zorder>btnTwo</zorder>
   <zorder>btnQuit</zorder>
   <zorder>btnThree</zorder>
   <zorder>lineUsername</zorder>
   <zorder>linePassword</zorder>
   <zorder>btnLogin</zorder>
   <zorder>chooseTitle</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
